twitter-lm.css

```diff
@@ -0,0 +1,491 @@
+  color: lightblue;
+  cursor: pointer;
+}
+
+.left-side .list-container .list-item:first-of-type i {
+  color: lightblue;
+}
+
+.left-side .list-container .list-item:hover i {
+  color: lightblue;
+}
+
+.left-side .list-container .list-item:last-of-type {
+  display: flex;
+  align-items: center;
+  margin-left: 0.2rem;
+  gap: 1rem;
+}
+
+.left-side .list-container .list-item:last-of-type .png {
+  background-image: url(../icons/more.png);
+  background-size: cover;
+  width: 3rem;
+  height: 3rem;
+  margin: 0rem 0.6rem 0rem 0.8rem;
+}
+
+.left-side .list-container .list-item:last-of-type:hover .png {
+  background-image: url(../icons/more-hover.png);
+  background-repeat: no-repeat;
+}
+
+.left-side .list-container .icon-wrapper {
+  display: inline-block;
+  width: 5rem;
+  display: inline-flex;
+  justify-content: center;
+}
+
+.left-side .profile {
+  padding: 1rem;
+  margin-bottom: 1rem;
+  align-items: center;
+  border-radius: 20rem;
+}
+
+.left-side .profile:hover {
+  background-color: lightgrey;
+  transition: all 0.3s;
+  cursor: pointer;
+}
+
+.center-wrapper-sm {
+  max-width: 96rem;
+  padding-left: 1.6rem;
+  padding-right: 1.6rem;
+  margin: 0 auto;
+}
+
+.center-wrapper-md {
+  max-width: 140rem;
+  padding-left: 1.6rem;
+  padding-right: 1.6rem;
+  margin: 0 auto;
+}
+
+.gap-sm {
+  gap: 0.4rem;
+}
+
+.gap-md {
+  gap: 0.8rem;
+}
+
+.gap-lg {
+  gap: 1.6rem;
+}
+
+.gap-lg-2x {
+  gap: 3rem;
+}
+
+.pd-lg {
+  padding: 1.6rem;
+}
+
+.cl-fade {
+  color: rgba(70, 68, 68, 0.664);
+}
+
+.tweet-btn {
+  width: 100%;
+  display: flex;
+}
+
+.tweet-btn #tweet-btn {
+  text-align: center;
+  border: transparent;
+  border-radius: 20rem;
+  color: white;
+  font-size: 2rem;
+  padding: 1.6rem 8rem;
+  background-color: lightgrey;
+}
+
+.tweet-btn #tweet-btn:hover {
+  background-color: cyan;
+  transition: all 0.3s;
+  cursor: pointer;
+}
+
+.twitter-links {
+    text-decoration: none;
+}
+
+.twitter-links2 {
+    text-decoration: none;
+    color: black;
+}
+
+.twitter-links3 {
+    text-decoration: none;
+    color: black;
+}
+
+.border-blue {
+  border: 2px dashed darkturquoise;
+  border-radius: 5px;
+
+  padding: 10px;
+  margin-left: 5px;
+  margin-right: 5px;
+}
+
+.tweet-option {
+  height: 100%;
+  align-items: center;
+  cursor: pointer;
+}
+
+.tweet-options {
+  grid-auto-rows: 24px;
+  display: grid;
+  grid-auto-flow: column;
+  position: relative;
+}
+
+.flex {
+  display: flex;
+}
+
+.gap-sm {
+  gap : 5px;
+}
+
+.tweet-container > .tweet .pd-md .b-b-thin {
+  border: solid 5px lightgrey;
+}
+
+.tweet-container {
+  border: 3px solid lightgrey;
+  border-radius: 5px;
+}
+
+.right-title {
+  margin-top: 20px;
+  margin-left: 20px;
+}
+
+.right-side {
+  width: 33rem;
+  padding: 0.6rem 1.2rem 0rem 1.2rem;
+  margin-bottom: 1.2rem;
+}
+
+.right-side .searchbar {
+  position: relative;
+}
+
+.right-side .searchbar i {
+  position: absolute;
+  font-size: 2rem;
+  left: 24px;
+  top: 50%;
+  transform: translateY(-50%);
```

Comment:
I don't know this either.

Reply:
a transform element is used if you want to align/centre an element with its parent - transform: translateY(-50%) in this case means "move me upwards by 50% of my height, along the y-axis"